dnsdbq.c

```diff
@@ -1063,7 +1057,12 @@ do_batch(FILE *f, qparam_ct qpp) {
 				io_engine(MAX_JOBS);
```

Comment:
Maybe rename MAX_JOBS to MAX_FETCHES to be consistent with language like "... start one or more curl fetches"